regression/cbmc-java/NegativeArraySizeException/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+NegativeArraySizeException.class
+--java-throw-runtime-exceptions
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
```

Comment:
It would be nice if the test could test more than this - I don't know if this is possible on CBMC, but something that actually validates that CBMC has figured out a run time exception should be thrown